toolkit/docs/building/prerequisites-ubuntu.md

```diff
@@ -28,8 +26,9 @@ sudo apt -y install \
     wget \
     xfsprogs
 
-# Fix go 1.19 link
-sudo ln -vsf /usr/lib/go-1.19/bin/go /usr/bin/go
+# Fix go 1.20 link
```

Comment:
Do we still need to do this ?
I am wondering that `Ubuntu` should do the right thing by default.

Reply:
My test for this is:
1. Install Go.
2. See if `go version` does anything.

It didn't find it, so for now we rely on this manual step.:)